js/BookList.js

```diff
@@ -1,3 +1,196 @@
+/* eslint max-classes-per-file: ["error", 2] */
+let bookList;
+const bookUl = document.querySelector('.book-list');
+
+class Node {
+  constructor(title, id, author, nextNode = null) {
+    this.title = title;
+    this.id = id;
+    this.author = author;
+    this.nextNode = nextNode;
+  }
+
+  add(title, id, author) {
+    if (this.nextNode === null) {
+      this.nextNode = new Node(title, id, author, null);
+    } else {
+      this.nextNode.add(title, id, author, null);
+    }
+  }
+
+  get(index, count) {
+    if (index === count) {
+      return this.title;
+    } if (this.nextNode !== null) {
+      return this.nextNode.get(index, count + 1);
+    }
+    return null;
+  }
+
+  getNode(index, count) {
+    if (index === count) {
+      return this;
+    } if (this.nextNode !== null) {
+      return this.nextNode.get(index, count + 1);
+    }
+    return null;
+  }
+
+  removeNode(index, count, old) {
+    if (index === count) {
+      old.nextNode = this.nextNode;
+      return true;
+    } if (this.nextNode !== null) {
+      return this.nextNode.remove(index, count + 1, this);
+    }
+    return null;
+  }
+
+  removebyId(id, old) {
+    if (this.id === id) {
+      old.nextNode = this.nextNode;
+      return true;
+    } if (this.nextNode !== null) {
+      return this.nextNode.removebyId(id, this);
+    }
+    return null;
+  }
+
+  addAt(index, count, value, old) {
+    if (index === count) {
+      const node = new Node(value, this);
+      old.nextNode = node;
+      return true;
+    } if (this.nextNode !== null) {
+      return this.nextNode.addAt(index, count + 1, value, this);
+    }
+    return null;
+  }
```

Comment:
- [ ] After adding second book when I saw the localstorage it gave me same data. Below is the screenshot. Also, when I add second book and refresh the website both book become same and when try to remove any of them and then refresh then both book gone. Please fix this issue. ( Make sure every time you add a new book it doesn't change the previous book id). :blush: 

### After adding second book (See the Console :point_down: )

![Screenshot from 2021-07-06 18-57-35](https://user-images.githubusercontent.com/22434660/124608970-d5483900-de8c-11eb-96c6-925a97a7cce5.png)


### After adding second book and refreshing


![Screenshot from 2021-07-06 18-58-04](https://user-images.githubusercontent.com/22434660/124609081-edb85380-de8c-11eb-916f-2241d0b3165e.png)